### PR TITLE
Argument parsing, and output format selection.

### PR DESCRIPTION
**Pull request by TroyDowling**
Just a few small features implemented to make performance better and the tool easier to use.

### CODE REVIEW

File: redBullFetch.py [hunk 1]
```diff
@@ -39,8 +43,9 @@ def generateScreenCaps(avFile, fps=3,
             raise OSError('Need an empty directory to generate screencaps!')
 
     # Generate screencaptures using ffmpeg
+    # Reference: https://ffmpeg.org/ffmpeg-filters.html#fps
     subprocess.run(['ffmpeg', '-i', './' + avFile, '-vf', 'fps=' + str(fps),
-        screenDir + '/img%05d.bmp'])
+        screenDir + '/img%06d.'+outType.lower()])
```
Comment (mwiens91) [inline redBullFetch.py:48]: The video has 16920 frames. Is there a good reason to allow for more than that?

**Comment by mwiens91** [comment]
Looks good! Thanks for the work and cleaning up the whitespace I left behind. The argparser library is interesting—I'll definitely have to look into it more. I'll merge these changes and update the readme to reflect them.

File: redBullFetch.py (hunk 1 above)
Comment (TroyDowling) [reply redBullFetch.py:48]: The only reason would be oversampling. If the tool is always used properly, then five digits is enough. But in an oversampling condition the namespace would deplete and FFMPEG would continuously overwrite the last filename. Adding another digit just adds a little wiggle room and doesn't cost anything.

**Comment by TroyDowling** [comment]
Glad you learned something new, I did too. I worked a lot with OpenCV in my last degree, but Tesseract OCR was something new to me. Having a chance to play around with tesserocr was why I forked your script, but you should know (in the interest of saving time and effort), the hand-completed data file provided in Piazza by George Michael Lertzman-Lepofsky is more than good enough to complete the assignment.

MATLAB Results using the Lertzman-Lepofsky data set:

altitude
velocity
acceleration

File: redBullFetch.py (hunk 1 above)
Comment (mwiens91) [reply redBullFetch.py:48]: Fair.

**Pull request merged**